examples/ClientTests/CMakeLists.txt

```diff
@@ -342,21 +342,22 @@ if (DD4HEP_USE_GEANT4)
     COMMAND    "${CMAKE_INSTALL_PREFIX}/bin/run_test_ClientTests.sh"
     EXEC_ARGS  python ${ClientTestsEx_INSTALL}/scripts/Check_Air.py
     -geometry  ${ClientTestsEx_INSTALL}/compact/Check_Air.xml batch
+    REQUIRES   DDG4 Geant4
```

Comment:
This argument doesn't exist for dd4hep_add_test_reg
```suggestion
```

Reply:
That's odd. I don't recall writing that line. I suspect it slipped in due to something I did wrong with git. So yeah, it can go.

Reply:
Probably from the `git revert`